README.md

```diff
@@ -1,30 +1,27 @@
-# TripleTen Sprint 3 Project: Spots
+# TripleTen Project: Spots
 
```

Comment:
* Please, remove the **white square** from the liked `heart` https://snipboard.io/wsMy67.jpg